src/Microsoft.Health.ControlPlane.Core/AssemblyInfo.cs

```diff
@@ -4,5 +4,7 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Resources;
+using System.Runtime.CompilerServices;
 
 [assembly: NeutralResourcesLanguage("en-us")]
+[assembly: InternalsVisibleTo("Microsoft.Health.ControlPlane.CosmosDb")]
```

Comment:
Why do we need this?

Reply:
There is a ToIdentityProvider method on the assembly that is used by controlplane.